### DISCUSSION TITLE
Github Action "post setup" is very slow!?

### DISCUSSION THREAD

**Discussion by ziyuhuang123** [community/community#42691] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi! I am learning github action and find that the "post setup" setup step is very slow!
Like below: 
![image](https://user-images.githubusercontent.com/99854690/209629070-15124e14-0aa8-47ae-b4f8-e04c8fe41245.png)
My code is very simple just for learning.... You can check the code on https://github.com/Arsmart1/test_action

Anyone has suggestion? Thanks!!!

**Comment by ziyuhuang123** [comment]
I noticed it is because I am using my own host server....What can I do??? Thanks!!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @ziyuhuang123 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by airtower-luna** [comment]
There is no "Explore-GitHub-Actions" workflow run in the repository you linked, can you link the workflow log?

The post-action for `actions/setup-python` should be uploading the cache if one is configured, I assume something goes wrong with that.
